### PR TITLE
done

### PR DESCRIPTION
**Pull request by RicoPsych**
Help me

### CODE REVIEW

File: JoinDatasets.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import shutil
+import os
+
+main_dir = input()
+output = input()
+
+
+labels = ["Healthy","Powdery","Rust"]
+for label in labels:
+    for set in ["Test","Train","Validation"]:
```
Comment (kawson1) [inline JoinDatasets.py]: it should be in enum variable as labels

Comment (RicoPsych) [reply JoinDatasets.py]: spieprzaj dziadu

Comment (kawson1) [reply JoinDatasets.py]: jesteś u pani

**Review by kawson1 — APPROVED** [review]

**Review by Danzigerrr — APPROVED** [review]
a great code by a great man

**Pull request merged**